src/client/Microsoft.Identity.Client/Extensibility/AppTokenProviderParameters.cs

```diff
@@ -1,14 +1,10 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
```

Comment:
using System;

Reply:
Not sure I understand this comment. I just used the "remove unused usings" refactoring tool in VS.